app/controllers/catalog_controller.rb

```diff
@@ -484,6 +484,13 @@ def search_tips
     end
   end
 
+  def documents_list
+    search_service = Blacklight::SearchService.new(config: blacklight_config)
+    ids = params[:ids].present? ? params[:ids].split : []
+    @documents = ids.empty? ? [] : search_service.fetch(ids)
```

Comment:
Should we set a limit on the number of docs to fetch from Solr at once? Maybe 1000?

Reply:
The original approach did use 1000 as a cap, so we can do that here as well